### PR TITLE
 Workflows and dependencies updates

### PR DESCRIPTION
**Pull request by sbesson**
- update Gradle workflow to publish SNAPSHOTS and release
- bump org.openmicroscopy.org plugin to consume version 5.5.4

### CODE REVIEW

**Comment by sbesson** [comment]
Briefly discussed the failure above with @jburel. Both this repository as well as https://github.com/ome/omero-blitz will require an extension of the workflow to support the installation of the `slice2java/slice2py` prerequisites.

**Comment by jburel** [comment]
The action-workflow run on all 3 os. The gradle workflow in this repo only works on ubuntu due to ice installation.
OS needs to be added as a parameter

**Comment by sbesson** [comment]
If I understand correctly, the proposal would be for this repository to consume something like the following?

```
    uses: ome/action-workflows/.github/workflows/gradle_build.yml@v1
    with:
      os: [ubuntu-20.04]
      install_ice: True
 ```

**Comment by jburel** [comment]
something like that.

File: .github/workflows/gradle.yml [hunk 1]
```diff
@@ -9,11 +9,11 @@ on:
 
 jobs:
   build:
-    uses: ome/action-workflows/.github/workflows/gradle_build.yml@v1
+    uses: jburel/action-workflows/.github/workflows/gradle_build.yml@d673bc75ec9185c679c1e89c174f6f86c10985d2
```
Comment (jburel) [inline .github/workflows/gradle.yml:12]: you need     ``with:
      install-ice: "true"`` here

**Comment by jburel** [comment]
https://github.com/ome/action-workflows/releases/tag/v1.2 is now available

**Review by jburel — APPROVED** [review]

**Pull request merged**